webknossos-datastore/app/com/scalableminds/webknossos/datastore/explore/RemoteLayerExplorer.scala

```diff
@@ -29,11 +25,7 @@ trait RemoteLayerExplorer extends FoxImplicits {
   def name: String
 
   protected def parseJsonFromPath[T: Reads](path: VaultPath): Fox[T] =
-    for {
-      fileBytes <- path.readBytes().toFox
-      fileAsString <- tryo(new String(fileBytes, StandardCharsets.UTF_8)).toFox ?~> "dataset.explore.failed.readFile"
-      parsed <- JsonHelper.parseAndValidateJson[T](fileAsString)
-    } yield parsed
+    path.parseAsJson[T]
```

Comment:
looks like this can be inlined now

Reply:
What do you mean by that?

Reply:
The function is a single line now, so I would suggest deleting the function and replacing all usages of parseJsonFromPath(path) directly by path.parseAsJson